test/e2e_node/garbage_collector_test.go

```diff
@@ -257,7 +257,10 @@ func containerGCTest(f *framework.Framework, test testRun) {
 
 // Runs containerGCTest using the docker runtime.
 func dockerContainerGCTest(f *framework.Framework, test testRun) {
-	runtime := docker.ConnectToDockerOrDie(defaultDockerEndpoint, defaultRuntimeRequestTimeoutDuration)
+	var runtime docker.DockerInterface
+	BeforeEach(func() {
+		runtime = docker.ConnectToDockerOrDie(defaultDockerEndpoint, defaultRuntimeRequestTimeoutDuration)
```

Comment:
what happens if the tests are run in parallel? I'm wondering if this should just be moved into the getContainerNames function